src/platform/ESP32/OpenthreadLauncher.cpp

```diff
@@ -196,8 +196,18 @@ static void try_update_ot_rcp(const esp_openthread_platform_config_t * config)
 static void rcp_failure_handler(void)
 {
     esp_rcp_mark_image_unusable();
-    try_update_ot_rcp(s_platform_config);
-    esp_rcp_reset();
+    char internal_rcp_version[kRcpVersionMaxSize];
```

Comment:
Let's try `esp_rcp_reset()` first, RCP update or restart should be used only when `esp_rcp_reset()` can't recover it.